src/ecephys_etl/modules/align_timestamps/__main__.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from allensdk.brain_observatory.argschema_utilities import \
+    ArgSchemaParserPlus, \
+    write_or_print_outputs
+from ._schemas import InputParameters, OutputParameters
+from .barcode_sync_dataset import BarcodeSyncDataset
```

Comment:
I think we've stopped using relative imports, because it was causing trouble when some dev was using windows and some particular IDE.